<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Data Recorder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="grpBoxDataRecorderSettings">
     <property name="title">
      <string>Data Recorder Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_items"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpBoxDataRecorder">
     <property name="title">
      <string>Data Recorder</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="lbl_message_recorder">
          <property name="text">
           <string>&lt;Message&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_initialize">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblCurrentDataFile">
          <property name="text">
           <string>Current data file: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_data_filename">
          <property name="text">
           <string>&lt; none &gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="grpBoxTrajectoryGenerator">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Trajectory Generator</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QLineEdit" name="line_trajectory_name"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_trajectory_name">
        <property name="text">
         <string>Trajectory Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_generate_trajectory">
        <property name="text">
         <string>Generate and Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
